crates/uv/src/commands/tool/run.rs

```diff
@@ -386,8 +386,16 @@ pub(crate) async fn run(
         args.iter().map(|arg| arg.to_string_lossy()).join(" ")
     );
 
-    // Unblock cache removal operations.
```

Comment:
We'll need this same change in the project run command per https://github.com/astral-sh/uv/issues/15989